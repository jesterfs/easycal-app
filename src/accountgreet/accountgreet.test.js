import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import AccountGreet from './accountgreet'
import renderer from 'react-test-renderer';

describe('AccountGreet component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <AccountGreet />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <AccountGreet />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});