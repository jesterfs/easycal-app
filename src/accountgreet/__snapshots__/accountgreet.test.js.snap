// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountGreet component renders the UI as expected 1`] = `
<div
  className="accountgreetgroup "
>
  <div
    className="item"
  >
    <h2>
      Hi 
      !
    </h2>
  </div>
  <div
    className="item"
  >
    <div
      className="accountLine"
    >
      <h3>
        Full Name:
      </h3>
       
       
    </div>
    <div
      className="accountLine"
    >
      <h3>
        Email:
      </h3>
       
       
    </div>
    <div
      className="accountLine"
    >
      <h3>
        Calendars: 
      </h3>
      <ul />
    </div>
    <div
      className="accountBtn"
    >
      <a
        href="/changepassword"
        onClick={[Function]}
      >
        <button
          className="addbtn"
        >
          Change Your Password
        </button>
      </a>
    </div>
    <div>
      <button
        className="accountBtn addbtn"
        onClick={[Function]}
      >
        Log Out
      </button>
    </div>
  </div>
</div>
`;
